### ISSUE TITLE
SkyEmu GBA core

### ISSUE THREAD

**Issue by getCursorsExe** [TASEmulators/BizHawk#3311]
see https://github.com/skylersaleh/SkyEmu/issues/47
Has some issues, but they are tracked currently.
Unlike mGBA, this emulator passes almost all GBA test ROMs, which would allow accurate emulation to appear.

**Comment by CasualPokePlayer** [comment]
The emu is relatively young, and passing testroms is not the holy grail of emulators (it's more of a dick waving contest to use them for comparison sake, they are better used for regression testing). The code organization doesn't seem pretty either, most of the "core" is just a single .h file with most things lumped into that. Not exactly a pretty thing to port. Doesn't seem to be as featureful as mGBA either.

**Comment by nattthebear** [comment]
It does seem like it would be a low effort port, but for a system that we already have a core for we need more than that; we need a compelling reason to ever use it over the existing core.

**Comment by vadosnaprimer** [comment]
> It's currently tracked in upstream, so it might happen.

What does that even mean?

**Comment by nattthebear** [comment]
It means they bothered the upstream author too

**Comment by CasualPokePlayer** [comment]
I tried the core for a bit, and the first game I tried, Pokemon Emerald, ended up having noticably wrong sound effects (being anything using the GB PSG channels).

And a look at the issue tracker shows GBA bugs are starting to roll in. As to be expected with such a young emulator just released.

The talk on this being "better" than mGBA in accuracy frankly just reeks of marketing. The cherry picked "hard to emulate games" and all those testroms don't equate to being overall more accurate.

In time I imagine these issues will be fixed, but bottom line is this emulator needs time to mature before it can seriously be considered for Hawk.
